code/acting/src/acting/potential_field_node.py

```
@@ -0,0 +1,254 @@
+#!/usr/bin/env python
+from ros_compatibility.node import CompatibleNode
+
+from mapping.msg import Map as MapMsg
+from nav_msgs.msg import Path
+from geometry_msgs.msg import PoseStamped
```

Ruff (0.8.2)

`geometry_msgs.msg.PoseStamped` imported but unused

Remove unused import: `geometry_msgs.msg.PoseStamped`

(F401)